File: foxogram-common/src/main/java/su/foxogram/dtos/api/response/InfoDTO.java
package su.foxogram.dtos.api.response;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Schema(name = "Ok")
public class InfoDTO {

	private String version;

	private String cdnURL;

	private String gatewayURL;

	private String appURL;

	public InfoDTO(String version, String cdnURL, String gatewayURL, String appURL) {
		this.version = version;
		this.cdnURL = cdnURL;
		this.gatewayURL = gatewayURL;
		this.appURL = appURL;
	}
}

File: src/main/java/su/foxogram/dto/api/response/InfoDTO.kt
package su.foxogram.dto.api.response

import io.swagger.v3.oas.annotations.media.Schema
import lombok.Getter
import lombok.Setter

@Getter
@Setter
@Schema(name = "Info")
data class InfoDTO(
    val version: String?, val cdnURL: String?, val gatewayURL: String?, val appURL: String?
)
